flexwrfoutput/openfiles.py

```diff
@@ -0,0 +1,77 @@
+"""
+Functions to handle raw netCDF output of FLEXPART-WRF.
+
+Meant to be used before the postprocessing with the accessor.
+"""
+from pathlib import Path
+from typing import Tuple, Union
+
+import xarray as xr
+
+
+def combine(flxout: xr.Dataset, header: xr.Dataset) -> xr.Dataset:
+    """Combines dimensions of flxout with header to have full information of output in\
+        one xarray.
+
+    Args:
+        flxout (xr.Dataset): Loaded flxout file.
+        header (xr.Dataset): Loader header file.
+
+    Returns:
+        xr.Dataset: Combined Dataset
+    """
+    combined = xr.merge([flxout, header.drop_dims("Time")])
+    return combined
+
+
+class AmbiguousPathError(Exception):
+    pass
+
+
+def get_output_paths(path: Union[str, Path]) -> Tuple[Path, Path]:
+    """Finds header and flxout files in directory and returns their paths.
+
+    Args:
+        path (Union[str, Path]): Path of output directory of FLEXPART-WRF.
+
+    Returns:
+        Tuple[Path, Path]: (flxout path, header path)
+    """
+    path = Path(path)
+
+    header_files = list(path.glob("header*"))
+    flxout_files = list(path.glob("flxout*"))
+
+    if len(header_files) == 0:
+        raise (
+            FileNotFoundError(f"Did not find a header file in given directory {path}")
+        )
+    if len(flxout_files) == 0:
+        raise (
+            FileNotFoundError(f"Did not find a flxout file in given directory {path}")
+        )
+    if len(header_files) > 1:
+        raise (
+            AmbiguousPathError(f"Found multiple header files in given directory {path}")
+        )
+    if len(flxout_files) > 1:
+        raise (
+            AmbiguousPathError(f"Found multiple flxout files in given directory {path}")
+        )
```

Comment:
```suggestion
    if not (len(header_files) or len(flxout_files)):
        missing_file =  " or ".join([fname for fname,file_exists in [("header", len(header_files)), ("flxout", len(flxout_files))] if not file_exists])
        raise (
            FileNotFoundError(f"Did not find a {missing_file} file in given directory {path}")
        )
    elif len(header_files) > 1 or len(flxout_files) > 1:
        duplicate_filetype =  " and ".join([fname for fname,num_files in [("header", len(header_files)), ("flxout", len(flxout_files))] if num_files > 1])
        raise (
            AmbiguousPathError(f"Found multiple {duplicate_filetype} files in given directory {path}")
        )
```

Reply:
Done in ef91c69aaa6e30ec8cd93d19486c0a9aca735988